.lintrunner.toml

```diff
@@ -940,6 +940,25 @@ command = [
     '@{{PATHSFILE}}'
 ]
 
+[[linter]]
+code = 'CONTEXT_DECORATOR'
+include_patterns = [
+    'torch/**',
+]
+command = [
+    'python3',
+    'tools/linter/adapters/grep_linter.py',
+    '--pattern=@.+(dynamo_timed)',
```

Comment:
```suggestion
    '--pattern=@.*(dynamo_timed)',
```

Or you specifically don't want to capture plain `@dynamo_timed` ?

Reply:
I want to catch @torch._dynamo.utils.dynamo_timed() too

Reply:
Right, so you want the `*` version, not `+` ?

Reply:
oh whoops you right